src/common/src/util/epoch.rs

```diff
@@ -151,8 +156,8 @@ impl EpochPair {
     }
 
     pub fn inc(&mut self) {
-        self.curr += 1;
-        self.prev += 1;
+        self.prev = self.curr;
+        self.curr += EPOCH_INC_MIN_STEP_FOR_TEST;
     }
 
     pub fn inc_for_test(&mut self, inc_by: u64) {
```

Comment:
Just to confirm, does this one need to shift `inc_by`? Or is it even used? It'll be more intuitive to me if these two methods are `inc(&mut self)` and `inc_for(&mut self, n_steps: usize)`.